### ISSUE TITLE
Include option for 'muted' Tooltips (CANVAS-1549)

### ISSUE THREAD

**Issue by jpante** [Workday/canvas-kit#961]
## Background

Presently we have 3 types of tooltips either in Canvas Kit, or pending.

Label - takes the aria-label from a focusable control and displays it in a tooltip. The tooltip contents are not announced by a screen reader, as the aria-label acts as the label for the button.

```html
<button type="button" aria-label="Open">
  <svg focusable="false" aria-hidden="true">...</svg>
</button>
...
<div>
 Open
</div>
Describe - augments the accessible name of a focusable control by allowing a custom string to be shown in a tooltip. This custom string is announced after a delay by a screen reader.
<button type="button" aria-describedby="hint">
 Open
</button>
...
<div id="hint">
  This is hint text
</div>
```

Truncate - takes truncated text from a focusable control and displays it un-truncated inside a tooltip. The tooltip contents are not announced by a screen reader.

---

## 🚀 Feature **Proposal**

New feature designs are highlighting a desire for a 4th type of tooltip - a muted hint if you like, this would behave as follows -

Allows a custom string to be displayed in a tooltip on a focusable control when, 
that control already has an accessible name 
if the control has a state, this is conveyed to an assistive technology 
it could otherwise be inferred from the context what action the control will perform, the tooltip is a visual hint and not a substitute for a label.
The custom string in the tooltip is not exposed to the accessibility api (thus not announced by a screen reader).

**Example markup - elided**

```html
<button type="button" aria-label="Open panel" aria-expanded="false">
 <svg>...</svg>
</button>
...
<!-- Tooltip -->
<div aria-hidden="true">
 Show stuff
</div>
```

**Comment by NicholasBoll** [comment]
I found this needed in v4 and I was already working on it, so I moved to v4 in #980
